src/core/tests/eval.cpp

```diff
@@ -2655,6 +2656,1216 @@ TEST(eval, evaluate_fake_quantize_dynamic_input) {
+
+    auto result = ov::Tensor();
+    auto out_vector = ov::TensorVector{result};
+    auto in_vector = ov::TensorVector{make_tensor<et>(data_shape, input_data)};
+    ASSERT_TRUE(model->evaluate(out_vector, in_vector));
+    result = out_vector.at(0);
+
+    EXPECT_EQ(result.get_element_type(), et);
+    EXPECT_EQ(result.get_shape(), data_shape);
+    EXPECT_THAT(
+        read_vector<float>(result),
+        Pointwise(
+            FloatEq(),
+            std::vector<
+                float>{0.f, 0.1015625f, 0.203125f, 0.3125f, 0.40625f, 0.5f, 0.625f, 0.6875f, 0.8125f, 0.875f, 1.f}));
+}
+
+TEST(eval, evaluate_fake_convert_f32_seq_to_f8e4m3_scale_false) {
+    using namespace testing;
+    constexpr auto et = element::f32;
+
+    const auto data_shape = Shape{8};
+
+    std::vector<float> input_data;
+    std::generate_n(std::back_inserter(input_data), shape_size(data_shape), ov::SeqGen<float>(0.143f));
+
+    auto data = make_shared<ov::op::v0::Parameter>(et, data_shape);
+    auto scale = op::v0::Constant::create(et, Shape{1}, {1.f});
+    auto shift = op::v0::Constant::create(et, Shape{1}, {0.f});
+
+    auto op = make_shared<op::v13::FakeConvert>(data, scale, shift, "f8e4m3");
+    auto model = make_shared<Model>(OutputVector{op}, ParameterVector{data});
+
+    auto result = ov::Tensor();
+    auto out_vector = ov::TensorVector{result};
+    auto in_vector = ov::TensorVector{make_tensor<et>(data_shape, input_data)};
+    ASSERT_TRUE(model->evaluate(out_vector, in_vector));
+    result = out_vector.at(0);
+
+    EXPECT_EQ(result.get_element_type(), et);
+    EXPECT_EQ(result.get_shape(), data_shape);
+    EXPECT_THAT(read_vector<float>(result),
+                Pointwise(FloatEq(), std::vector<float>{0.140625, 1.125, 2.25, 3.25, 4, 5, 6, 7}));
+}
+
+TEST(eval, evaluate_fake_convert_f32_seq_to_f8e4m3_scale_1) {
+    using namespace testing;
+    constexpr auto et = element::f32;
+
+    const auto data_shape = Shape{8};
+
+    std::vector<float> input_data;
+    std::generate_n(std::back_inserter(input_data), shape_size(data_shape), ov::SeqGen<float>(0.143f));
+
+    auto data = make_shared<ov::op::v0::Parameter>(et, data_shape);
+    auto scale = op::v0::Constant::create(et, Shape{1}, {1.f});
+    auto shift = op::v0::Constant::create(et, Shape{1}, {0.f});
+
+    auto op = make_shared<op::v13::FakeConvert>(data, scale, shift, "f8e4m3");
+    auto model = make_shared<Model>(OutputVector{op}, ParameterVector{data});
+
+    auto result = ov::Tensor();
+    auto out_vector = ov::TensorVector{result};
+    auto in_vector = ov::TensorVector{make_tensor<et>(data_shape, input_data)};
+    ASSERT_TRUE(model->evaluate(out_vector, in_vector));
+    result = out_vector.at(0);
+
+    EXPECT_EQ(result.get_element_type(), et);
+    EXPECT_EQ(result.get_shape(), data_shape);
+    EXPECT_THAT(read_vector<float>(result),
+                Pointwise(FloatEq(), std::vector<float>{0.140625, 1.125, 2.25, 3.25, 4, 5, 6, 7}));
+}
```

Comment:
I might have missed something, but those 2 test cases seem to be identical, similar for test  `evaluate_fake_convert_f32_to_f8e5m2_scale_false / evaluate_fake_convert_f32_to_f8e5m2_scale_` or others?